applications/machine_learning/README.rst

```diff
@@ -207,6 +207,9 @@ Buttons
 The application supports a button that is used to switch between data forwarding and running the machine learning model.
 You can trigger the change by pressing and holding the button for longer than 5 seconds.
 
+.. note::
+   If a given configuration supports Bluetooth, then two sequences of the button press and release in a short time interval occurring shortly after application boot trigger removing Bluetooth bond.
```

Comment:
```suggestion
   If a given configuration supports Bluetooth, then button press and release in a short time interval, shortly after the application boot trigger, removes the Bluetooth bond.
```

Reply:
This is not true now. We need to perform double click (`two sequences of the button press and release in a short time interval`)

Reply:
I meant that the 2 sequences should appear shortly after application is booted.